docs/configuration/layers/vector.md

```diff
@@ -136,6 +136,19 @@ This sample configuration styles features having a `Charging_Level` value of 3 o
 ]
 ```
 
+## Legend Property
+`"legend": Object`
+
+Optional attributes to configure the display of the layer in legends.
+
+`"point", "line", "fill": Boolean`
+
+Set the shape of the legend swatch. "line" is the default. 
+
+`"includeOtherLegendWithDefaultStyling": Boolean`
```

Comment:
On further thought, I think I will reverse this to something like "hideOtherLegendWithDefaultStyling". I think we should include the default styling as a ... default :) . I think most uses of conditional styling will want to include the default, and I want to favour inclusion.